hack/init-environment.sh

```diff
@@ -239,8 +241,8 @@ function install_cloudsql_proxy () {
     --version=0.0.5 \
     --set secrets[0].secretName=cloudsqlkey \
     --set secrets[0].vals[0].kubeSecretKey=cloudsqlkey.json \
-    --set secrets[0].vals[0].path=$vault_location \
-    --set secrets[0].vals[0].vaultKey=proxy_account_key
+    --set secrets[0].vals[0].path=vault_sa_location \
```

Comment:
vault_sa_location is a variable, right?
```suggestion
    --set secrets[0].vals[0].path=$vault_sa_location \
```

Reply:
THANK YOU yes